src/components/email/Email.tsx

```diff
@@ -226,3 +226,51 @@ export const pageQuery = graphql`
     }
   }
 `;
+
+const StyledSection = styled.section`
```

Comment:
our site doesn't really need media queries anymore. Outside of a few edge-cases, we can just use basic calculations to output our layout